src/hooks/init.js

```diff
@@ -20,5 +21,19 @@ module.exports = async context => {
     process.exit() // eslint-disable-line
   }
 
+  if (process.argv.length > 3 &&
+      ['-v', '--version', 'version'].includes(process.argv[2]) && process.argv[3] === '--verbose') {
```

Comment:
Should the order matter?

```
if (process.argv.includes(['-v', '--version', 'version'] && process.argv.includes('--verbose')) {
```

Reply:
Yes we want to make sure it is the `netlify version` command because this logic runs on everything.

Reply:
Another command could have `version` flag as well. We only want to run this logic specifically for `netlify version`